tidalapi/user.py

```diff
@@ -380,3 +383,18 @@ def videos(self) -> List["Video"]:
                 f"{self.base_url}/videos", parse=self.session.parse_media
             ),
         )
+
+    def mixes(self, limit: Optional[int] = 50, offset: int = 0) -> List["MixV2"]:
+        """Get the users favorite tracks.
+
+        :return: A :class:`list` of :class:`~tidalapi.media.Track` objects containing all of the favorite tracks.
+        """
+        params = {"limit": limit, "offset": offset}
+        return cast(
+            List["MixV2"],
+            self.requests.map_request(
+                url=urljoin("https://api.tidal.com/v2", f"{self.v2_base_url}/mixes"),
```

Comment:
Doing a simple test results in a 404 HTTP Error
```
mixes = self._active_session.user.favorites.mixes()
...
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.tidal.com/favorites/mixes?sessionId=XXX&countryCode=DK&limit=50&offset=0
```
A quick glance reveals that v2 is omitted in the url so V2 url is invalid.

urljoin removes the `v2` part. It looks like we need to add a forward slash. Adding it fixes the issue.

```suggestion
                url=urljoin("https://api.tidal.com/v2/", f"{self.v2_base_url}/mixes"),
```

Reply:
Confused, I'm certain that test passed on my machine when I was running it.

Reply:
Yeah its quite odd. Maybe a different version of urllib?